docs/static/releasenotes.asciidoc

```diff
@@ -76,12 +76,43 @@ This section summarizes the changes in the following releases:
 === Logstash 8.17.0 Release Notes
 
 [[known-issues-8-17-0]]
-==== Known issue
+==== Known issues
+
+[[known-issue-8-17-0-jvm]]
+===== JVM version changes needed when upgrading Logstash from 8.12.0 (or earlier)
+
+If the `jvm.options` file was modified and not overwritten with the newest version, you may see a "deserialize invocation error" message, causing the pipeline to crash. 
+Users are affected if the Persistent Queue (PQ) is enabled, and the pipeline is processing messages larger than 20MB.
+
+**Solution:** Apply the default change contained in the newer 'jvm.options' file, as seen in this https://github.com/elastic/logstash/blob/main/config/jvm.options#L66-L82[example].
+
+[[known-issue-8-17-0-json_lines]]
+===== Error with `json_lines` codec 3.2.0 (or earlier) and {ls} 8.17.1 (or earlier)
```

Comment:
```suggestion
===== Error with `json_lines` codec 3.2.0 (or later) and {ls} 8.17.1 (or earlier)
```
Based on description in the issue, I think this should be 3.2.0 or later for json_lines?

Reply:
Yup, `json_lines codec >= 3.2.0` versions which have an ability to set the buffer limit.